config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,22 @@
+CarrierWave.configure do |config|
+
+  if Rails.env.test?
+    config.enable_processing = false
+  end
+
+  if Rails.env.development? || Rails.env.test?
+    config.storage = :file
+  else
+    config.storage = :fog
+    config.fog_credentials = {
+      provider:              'AWS',
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.